src/Social/Messages/Repositories/MessageRepository.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Kanvas\Social\Messages\Repositories;
+
+use Kanvas\Social\Messages\Models\Message;
+
+class MessageRepository
+{
+    /**
+     * getById
+     */
+    public static function getById(int $id): Message
```

Comment:
@FredPeal use https://github.com/bakaphp/kanvas-ecosystem-api/blob/development/src/Baka/Traits/SearchableTrait.php#L12 , no?

Reply:
the trait search by company, in this table I don't use companies_id